samples/components/dialects/Annotated.tdd

```diff
@@ -1006,7 +1008,8 @@
       <argument type='datetime' />
     </date-function>
     <date-function name='DATEPART' return-type='int'>
-      <formula>FLOOR((7 + TO_NUMBER(TO_CHAR(%2,'DDD')) - 1 + MOD(7 + MOD(TO_NUMBER(TO_CHAR(TRUNC(CAST(%2 AS DATE), 'YYYY'), 'J')) + 1, 7) - %3, 7)) / 7)</formula>
+      <formula>TO_NUMBER(TO_CHAR(%2,&apos;%1&apos; %3&apos;))</formula>
```

Comment:
remove `<formula>TO_NUMBER(TO_CHAR(%2,&apos;%1&apos; %3&apos;))</formula>`, only need `<formula part='week'> ...` for custom SOW paths.

Same comment for others in this file and full_dialect.tdd